src/main/java/org/jenkinsci/plugins/pagerduty/changeevents/ChangeEvent.java

```diff
@@ -35,7 +32,7 @@ public class ChangeEvent {
     /**
      * Additional information about the change.
      */
-    private HashMap<String, ?> customDetails;
+    private Map<String, ?> customDetails;
```

Comment:
I updated the type here so I could more easily populate this after parsing user input with Jackson.  Didn't see any usage that was specifically requiring HashMap vs the interface.